dcm2niix/dcm2niix_run.py

```diff
@@ -126,6 +133,37 @@ def convert_directory(
             # See: https://www.nitrc.org/forum/forum.php?thread_id=11134&forum_id=4703
             converter.inputs.series_numbers = convert_only_series
 
+        # dcm2niix command configurations for: crop
```

Comment:
I like how you expanded this section to include comments for the dcm2niix converter options.

Reply:
:+1: